plugins/hls-refine-imports-plugin/src/Ide/Plugin/RefineImports.hs

```diff
@@ -195,13 +195,20 @@ refineImportsRule = define $ \RefineImports nfp -> do
   -- for modules/symbols actually got used
   (imports, mbMinImports) <- liftIO $ extractMinimalImports hsc tmr
 
+  -- Internal module is the convention that no module should import
+  -- directly
+  let notContainInternalModule :: [AvailInfo] -> Bool
+      notContainInternalModule = not . any (\a ->
+        "Internal" `isSuffixOf` prettyPrint (availName a))
```

Comment:
It shouldn't be too hard to add configuration for the internal suffix - @berberman and @isovector are the experts

Reply:
FYI, an example of  `Properties` in type lenses plugin:
https://github.com/haskell/haskell-language-server/blob/7d7cfa24c0d8efa2188ac1753cbf9cd74a310672/ghcide/src/Development/IDE/Plugin/TypeLenses.hs#L89-L112

Reply:
@rayshih had the example above helped to implement the config of the internal suffix?